develop/develop/settings.py

```diff
@@ -148,7 +148,7 @@
 # STYLIST_DEFAULT_CSS = STATIC_URL + "css/default.css"
 STYLIST_DEFAULT_CSS = os.path.join(MEDIA_ROOT, "css/default.css")
 STYLIST_SCSS_TEMPLATE = os.path.join(BASE_DIR, "scss/base_template.scss")
-STYLIST_IGNORE_SASS = False
+STYLIST_IGNORE_SASS = True
```

Comment:
i think this caused some tests to fail; it's ok to set it to false for testing purposes, i just think the default should be True when users download the package

Reply:
ah yes.